src/bionics_ui.cpp

```diff
@@ -142,7 +142,7 @@ void draw_description( WINDOW *win, bionic const &bio )
     ypos += 1 + fold_and_print( win, ypos, 0, width, c_ltblue, bionic_info( bio.id ).description );
 
     // @todo Unhide when enforcing limits
-    if( g->u.has_trait( "DEBUG_CBM_SLOTS" ) ) {
+    if( g->u.has_trait( trait_id("DEBUG_CBM_SLOTS") ) ) {
```

Comment:
Styling tip: add space inside the brackets, like this:
```c++
if( g->u.has_trait( trait_id( "DEBUG_CBM_SLOTS" ) ) ) {
```
Otherwise the astyle check won't pass.